### ISSUE TITLE
Repository showing as excluded project

### ISSUE THREAD

**Issue by dhruvrkashyap** [aryangulati/Hacktoberfest2021#219]
In hacktoberfest's website, this repository is showing as excluded project, so the pull request made are not accepted.
Kindly fix this error.

**Comment by aryangulati** [comment]
Yes I am making new repo for Hacktoberfest21 with more exciting Projects
